.github/workflows/rpm.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         env:
           _VERSION_ : ${{ steps.date.outputs.date }}
```

Comment:
does it make more sense to use git commit for version?
The Makefile already sets the correct git sha as version https://github.com/sustainable-computing-io/kepler/blob/main/Makefile#L15
This `VERSION` is used to build kepler. 


```
Kepler running on version: v0.6.1-100-g0fd8a5d-dirty
```
so we already have correct version info, just need to use it.

Reply:
good point. since no '-' is allowed in `VERSION`, I am using `.` now